src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -209,6 +209,19 @@ private async Task CreateStandbyWarmupFunctions()
             File.WriteAllText(Path.Combine(functionPath, "run.csx"), content);
 
             _logger.LogInformation("StandbyMode placeholder function directory created");
+
+            ReadRuntimeAssemblyFiles();
+        }
+
+        private void ReadRuntimeAssemblyFiles()
+        {
+            string assemblyLocalPath = Path.GetDirectoryName(new Uri(typeof(StandbyManager).Assembly.Location).LocalPath);
```

Comment:
To be super safe, let's put the entire method as well as reading individual file bytes in try/catch and log exceptions if they throw but continue for the rest.

Reply:
done!